packages/neuron-ui/src/services/remote/apiMethodWrapper.ts

```diff
@@ -42,13 +35,18 @@ export const controllerMethodWrapper = (controllerName: string) => (
   if (!window.remote) {
     return RemoteNotLoadError
   }
-  const controller = window.remote.require(`./controllers/${controllerName}`).default
+  const controller = window.remote.require('./controllers/api').default
   if (!controller) {
```

Comment:
The validation should include the error of `./controllers/api is not found`

Reply:
What did you mean by including the error of that? I moved that `const` of controller not loadable to inner object and return that.

BTW `RemoteNotLoadError` was not used as well - but it's exported.